### ISSUE TITLE
Add a readme describing installation and usage

### ISSUE THREAD

**Issue by agizmo** [Feuermurmel/youtube-podcast-gateway#1]
I have everything setup and can start the application ("Starting server on port 8080 ..."), but I don't know how to generate a RSS feed. Could you provide some guidance? I'm not very good and reading python code.

This is one of the few applications I've seen on github that works with YouTube's v3 API and I'm hoping it works with a personal project.

**Comment by Feuermurmel** [comment]
You mean, like, writing a short manual? ;) Yeah, thats missing right now. I'll add a readme.

Please note that you'll need to have a Google account to create an application key from the Google Developer Console and also to allow the application read-access to your account. I'll write the how and why in the upcoming readme.

**Comment by Feuermurmel** [comment]
@agizmo Can you try with the new readme? Also do a _pull_ and re-run `./setup-venv.sh`. I added `youtube-dl` to the installed dependencies.

**Comment by agizmo** [comment]
Worked like a charm. I had everything figured out except subscribing.

My Project: The last firmware update for the Tivo Premiere rendered the built-in Youtube app unusable. Playback was stuttering and audio would drop out.

A few years ago a member of the Tivo community wrote an application that would allow playback of RSS feeds on the Tivo so I figured if I could get an iPad compatible feed of YouTube videos then I could play them back on the Tivo. 

Your application is the only one I found on GitHub that can create a usable RSS feed that serves mp4 videos.  I took the URL for the generated feed and am now watching a YouTube video on my Tivo beautifully.

I really appreciate this application you wrote.
